packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -87,6 +88,20 @@ export class LayeredRuntimeResolverCreator {
         }
       }
 
+      // If delta marker is encountered, add resolver for `delta` query
+      if (parseMarker('delta', resolverElement.graphqlType.description)) {
+        const diffQuery = `get${resolverElement.graphqlType.name}Delta`
+
+        resolvers.Query[diffQuery] = (parent: any, args: any, context: any) => {
+          const filter: any = {};
+          if (args?.lastSync) {
```

Comment:
```suggestion
```

Since `lastSync` is a required argument, it is safe to assume `args` and `lastSync` are defined, as GraphQL will validate this client side.

Reply:
I actually have made it optional here:
https://github.com/aerogear/graphback/blob/72cc030a1fb4a190d1295b67def411289b324fcd/packages/graphback-datasync/src/DatasyncPlugin.ts#L85-L87

I was thinking if it could be used for getting data initially?

Reply:
It should be required for deltaQueries

Reply:
This is done now:
https://github.com/aerogear/graphback/blob/c5783e234d8c30f6e6bed1284223b14d44c95fc5/packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts#L94-L99
https://github.com/aerogear/graphback/blob/c5783e234d8c30f6e6bed1284223b14d44c95fc5/packages/graphback-datasync/src/DatasyncPlugin.ts#L78